### ISSUE TITLE
Ulqiorra mana nerf

### ISSUE THREAD

**Issue by tpdjairks** [FlyOutBoy/BvO-Another#15]
Nerf Ulqiorra max mana -340 current value.

Wait.. I'm so bad lol.

40 int so 40 x 12 = 480 mana from int. So base mana 210 + 480 = 690

So.. for base mana = 60 like all other champs. I think? And maybe + 30 int let's say 

Then 60 base mana + (30x12) = 420. Yeah let's go with this, that's more in line with the rest of the heroes and I don't want him to spam those spears too much to begin with.

So to recap:
-nerf base mana to 60 
-nerf base int to 30

**Comment by tpdjairks** [comment]
bump, edited.

**Comment by FlyOutBoy** [comment]
done
